src/heuristicblocking.js

```diff
@@ -181,9 +204,6 @@ HeuristicBlocker.prototype = {
     if(firstParties.indexOf(page_origin) === -1){
       firstParties.push(page_origin);
       snitch_map.setItem(tracker_origin, firstParties);
-      // TODO removing this line/messing up parameters should break integration tests
-      this.storage.setupHeuristicAction(tracker_fqdn, constants.ALLOW);
-      this.storage.setupHeuristicAction(tracker_origin, constants.ALLOW);
```

Comment:
This happens the first time we see a tracker (third party resource that had `hasTracking` return true) on a given page origin. Which seems kind of funky: doesn't this only need to happen the very first time we see a potential tracker, not once per page origin?

(Also what if it's a new origin for a known tracker? Then we reset the tracker's status (everywhere) to ALLOW? Well, no, since we should never get this far because of the `if (action != constants.NO_TRACKING && action != constants.ALLOW)` checks in calling functions.)

Seems like we should restore, plus document this logic, and fix or add a note to fix when it runs. My TODO comment is about it not having automated tests.